symphony-utility/src/main/java/org/symphonyoss/symphony/bots/utility/message/EntityBuilder.java

```diff
@@ -22,14 +24,32 @@ private EntityBuilder(String type, String version) {
     this.content.put("version", version);
   }
 
+  /**
+   * Create a new entity without type and version.
```

Comment:
This text may suggest that the `type` and the `version` are required.
If they are not, maybe the text _Create a new empty entity_ should be better